docs/dev/index.rst

```diff
@@ -14,6 +14,7 @@ Development
   datastore
   clone_ferc1
   existing_data_updates
+  run_a_release
```

Comment:
I added the release instructions to the toctree here, otherwise they don't show up anywhere and Sphinx complains.

Reply:
thank you!